src/layout/section/faqSection/FaqSection.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import uniqueId from 'lodash.uniqueid'
+
+import { Button } from '../../../button'
+import { ContentDivider } from '../../../divider/contentDivider'
+import { SpacingDivider, SpacingDividerSize } from '../../../divider/spacingDivider'
+import { SectionContentSize } from '../../../layout/section/baseSection'
+import { Paragraph } from '../../../paragraph'
+import { StyledFaqSection } from './FaqSection.style'
+
+export type FaqItemProps = {
+  question: string
+  answer: string
+}
+
+export type FaqSectionProps = Readonly<{
+  className?: string
+  title?: string
```

Comment:
The `title` prop should concern `title` html attribute to not conflict with a11y enhancements.
Th goal will be to add the `title` prop to `A11yProps` interface and be able to use it on every components.
Here is a pending PR [1113](https://github.com/blablacar/ui-library/pull/1113) on this topic.
@gabrielagabriel could we validate a prop name for main headings / main info in components like: 
`mainTitle`, `mainText`, `mainInfo`, `sectionHeadingText` or `mainInfoText`...

Reply:
Done, used `mainTitle`

Reply:
Ok thanks, it's also what I une on my PR.